packages/strapi-plugin-upload/admin/src/containers/InputModalStepperProvider/reducer.js

```diff
@@ -145,6 +145,12 @@ const reducer = (state, action) =>
         draftState.selectedFiles = draftState.selectedFiles.filter(file => file && file.id);
         break;
       }
+      case 'ON_SUBMIT_EDIT_NEW_FILE': {
```

Comment:
Can you add the related test for this case?

Reply:
test added.